packages/react-components/react-toolbar/src/stories/Toolbar/ToolbarWithPopover.stories.tsx

```diff
@@ -4,40 +4,106 @@ import type { ToolbarProps } from '@fluentui/react-toolbar';
 import { Popover, PopoverSurface, PopoverTrigger, Button } from '@fluentui/react-components';
 import { CalendarMonthRegular, Accessibility24Filled } from '@fluentui/react-icons';
 
-export const WithPopover = (props: Partial<ToolbarProps>) => (
-  <Toolbar {...props} size="small">
-    <Popover withArrow>
-      <PopoverTrigger disableButtonEnhancement>
-        <ToolbarButton appearance="primary">See more...</ToolbarButton>
-      </PopoverTrigger>
-      <PopoverSurface>
-        <div>
-          <h3>Popover content</h3>
-          <Button>Action</Button>
-        </div>
-      </PopoverSurface>
-    </Popover>
-    <ToolbarDivider />
-    <Popover withArrow>
-      <PopoverTrigger disableButtonEnhancement>
-        <ToolbarButton aria-label="Popover trigger - Calendar" icon={<CalendarMonthRegular />} />
-      </PopoverTrigger>
-      <PopoverSurface>
-        <div>
-          <h3>Popover content</h3>
-          <Button>Action</Button>
-        </div>
-      </PopoverSurface>
-    </Popover>
-    <Popover withArrow>
-      <PopoverTrigger disableButtonEnhancement>
-        <ToolbarButton aria-label="Popover trigger - Accessibility" icon={<Accessibility24Filled />} />
-      </PopoverTrigger>
-      <PopoverSurface>
-        <div>
-          <h3>Popover content - Accessibility</h3> <Button>Action</Button>
-        </div>
-      </PopoverSurface>
-    </Popover>
-  </Toolbar>
-);
+export const WithPopover = (props: Partial<ToolbarProps>) => {
+  const [open, setOpen] = React.useState({
```

Comment:
This would make more sense to be a string enum
```tsx
const [open, setOpen] = React.useState<'first' | 'second' | 'third' | undefined>(undefined);

<Popover open={open === 'first'} onOpenChange={(_, data) => setOpen(data.open ? 'first' : undefined)} />
```

Reply:
But that would make it only possible to have 1 opened per time

Reply:
Done https://github.com/microsoft/fluentui/pull/25635/commits/9ade33a69fd97b84abdba715ff0137837f311525

Reply:
Ah yeah, if that's what you're going for, it makes sense